dev-haskell/postgresql-simple/postgresql-simple-0.6.4.ebuild

```diff
@@ -24,10 +24,11 @@ RDEPEND=">=dev-haskell/aeson-1.4.1.0:=[profile?] <dev-haskell/aeson-1.6:=[profil
 	>=dev-haskell/fail-4.9.0.0:=[profile?] <dev-haskell/fail-4.10:=[profile?]
 	>=dev-haskell/hashable-1.2.7.0:=[profile?] <dev-haskell/hashable-1.4:=[profile?]
 	>=dev-haskell/only-0.1:=[profile?] <dev-haskell/only-0.1.1:=[profile?]
-	>=dev-haskell/postgresql-libpq-0.9.4.2:=[profile?] <dev-haskell/postgresql-libpq-0.10:=[profile?]
+	>=dev-haskell/postgresql-libpq-0.9.4.3:=[profile?] <dev-haskell/postgresql-libpq-0.10:=[profile?]
```

Comment:
Since the `RDEPEND` has changed, you probably need to bump the revision on the file.

I think an easy way to do this is `git mv postgresql-simple-0.6.3.ebuild postgresql-simple-0.6.3-r1.ebuild`

This way, portage recognizes that it needs to rebuild the package for anyone that already has it installed.

Using `git mv` helps to keep the diffs readable.

Reply:
The version is bumped entirely (0.6.3 -> 0.6.4), unless I am missing something here, so there is no need for a revbump. Portage will detect version bump and thus interpret new `RDEPENDS`.

Reply:
🤦‍♂️ if that's the case, then ofc. I guess I was relying on the diff, and saw just a  few lines changes, so I thought the version wasn't changed.